app/gui/view/component-browser/component-group/src/multi.rs

```diff
@@ -0,0 +1,212 @@
+//! Provides a multi component group wrapper, an abstraction used to propagate FRP events from
+//! multiple component groups.
+//!
+//! See [`Wrapper`] docs.
+
+
+use ensogl_core::prelude::*;
+
+use enso_frp as frp;
+use ensogl_core::display::scene::Scene;
+
+use crate::entry;
+use crate::wide;
+use crate::View;
+
+
+
+// =============
+// === Group ===
+// =============
+
+/// One of the possible component group types.
+#[derive(Debug, Clone, CloneRef)]
+#[allow(missing_docs)]
+pub enum Group {
+    Regular(View),
+    Wide(wide::View),
+}
```

Comment:
These docs do not explain what it is. After reading it, I don't know what is "Regular" and "Wide". This should be described in such a way, that the reader understands without reading implementation of methods later in this file.

Reply:
I renamed `Regular` -> `OneColumn`, but I'm unsure what else I should add. This is an enum with doc "One of the possible component group types." So by reading this doc, I understand that there are multiple component group types and they (or part of them) are listed in this enum. I can easily follow to the type I'm interested in. (`View` or `wide::View`)
I don't think it's wise to put all the information about component groups here. Let's chat about it if you disagree.